src/main/io/beeper.c

```diff
@@ -338,11 +340,20 @@ void beeperUpdate(timeUs_t currentTimeUs)
         return;
     }
 
+    #ifdef USE_DSHOT
+    if (!ARMING_FLAG(ARMED) && currentBeeperEntry->mode == BEEPER_RX_SET) {  
```

Comment:
Yeah I tried to use the dshot beep for everything, but that was a little too much.  I could see in the future you could use the 5 different kinds of beeps to signal different things much like the beeper has different patterns.

Also this has already saved me a few minutes finding my crashed quad in some tall grass!  I don't have actual beepers on any of my quads.